inlineplz/linters/__init__.py

```diff
@@ -7,11 +7,19 @@
 from __future__ import print_function
 
 import fnmatch
+from multiprocessing.pool import ThreadPool as Pool
 import os
 import subprocess
 import time
 import traceback
 
+# Use the built-in version of scandir/walk if possible, otherwise
+# use the scandir module version
+try:
+    from os import scandir, walk
```

Comment:
`prospector: pylint: Imports from package os are not grouped (ungrouped-imports)`